wd-java/README.md

```diff
@@ -21,8 +21,8 @@
 
 - Configure with your Sauce credentials from https://app.saucelabs.com/user-settings
   ```sh { name=set-credentials }
-  echo SAUCE_USERNAME=__YOUR_SAUCE_USERNAME__ >> src/test/resources/.env
-  echo SAUCE_ACCESS_KEY=__YOUR_SAUCE_ACCESS_KEY__ >> src/test/resources/.env
+  export SAUCE_USERNAME=__YOUR_SAUCE_USERNAME__
```

Comment:
To make this work, we'll need to provide an empty .env file so dotenv doesn't crash on startup

Reply:
or we do this when initializing dotenv:
```java
Dotenv
   .configure()
   .ignoreIfMissing()
   .load();
```